src/classes/CW_CampaignRewardTriggerHandler.cls

```diff
@@ -0,0 +1,30 @@
+public with sharing class CW_CampaignRewardTriggerHandler 
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.